backend/development.ini.sample

```diff
@@ -223,6 +253,10 @@ preview_cache_dir = %(basic_setup.preview_cache_dir)s
 
 ### Session ###
 
+# Warning ! if you decide to change the session parameters, we strongly encourage you to delete existing sessions
+# in order to avoid security issues. see tracim settings documentation (available in source
```

Comment:
```suggestion
# Warning! if you decide to change the session parameters, we strongly encourage you to delete existing sessions
# in order to avoid security issues. See the Tracim settings documentation (available in source
```